crates/fuel-core/src/database/transactional.rs

```diff
@@ -3,7 +3,7 @@ use crate::{
     state::in_memory::transaction::MemoryTransactionView,
```

Comment:
should the name of this file/module be renamed now that we no longer use `Transactional`?

Reply:
#896